libs/android/googleservices-build.gradle

```diff
@@ -0,0 +1,23 @@
+buildscript {
+    repositories {
+        jcenter()
```

Comment:
we should include first the mavenCentral instead of jcenter as we had in the past issues fetching data from jcenter